### PR TITLE
Update OpenRefine

### PR DESCRIPTION
**Pull request by RhinosF1**

### CODE REVIEW

**Comment by vivian-rook** [comment]
Openrefine does not seem to load off of this branch in the dev env. Eventually giving a `504 Gateway Time-out`

File: images/singleuser/Dockerfile [hunk 1]
```diff
@@ -166,7 +166,7 @@ USER root
 ## Setup OpenRefine
 ENV OPENREFINE_DIR /srv/openrefine
 RUN mkdir -p ${OPENREFINE_DIR} && cd ${OPENREFINE_DIR} && \
-    curl -L https://github.com/OpenRefine/OpenRefine/releases/download/3.5.2/openrefine-linux-3.5.2.tar.gz | tar xzf - --strip=1
+    curl -L https://github.com/OpenRefine/OpenRefine/archive/refs/tags/3.6.0.tar.gz | tar xzf - --strip=1
```
Comment (wetneb) [inline images/singleuser/Dockerfile:169]:
```suggestion
    curl -L https://oss.sonatype.org/service/local/artifact/maven/content?r=releases&g=org.openrefine&a=openrefine&v=3.6.0&c=linux&p=tar.gz | tar xzf - --strip=1
```
This is the new format for our download URLs.

**Comment by wetneb** [comment]
Sorry I probably got caught by the classical mistake of forgetting to escape the `&` in the URL.

File: images/singleuser/Dockerfile [hunk 2]
```diff
@@ -166,7 +166,7 @@ USER root
 ## Setup OpenRefine
 ENV OPENREFINE_DIR /srv/openrefine
 RUN mkdir -p ${OPENREFINE_DIR} && cd ${OPENREFINE_DIR} && \
-    curl -L https://github.com/OpenRefine/OpenRefine/releases/download/3.5.2/openrefine-linux-3.5.2.tar.gz | tar xzf - --strip=1
+    curl -L https://oss.sonatype.org/service/local/artifact/maven/content?r=releases&g=org.openrefine&a=openrefine&v=3.6.0&c=linux&p=tar.gz | tar xzf - --strip=1
```
Comment (wetneb) [inline images/singleuser/Dockerfile:169]:
```suggestion
    curl -L "https://oss.sonatype.org/service/local/artifact/maven/content?r=releases&g=org.openrefine&a=openrefine&v=3.6.0&c=linux&p=tar.gz" | tar xzf - --strip=1
```

Comment (RhinosF1) [reply images/singleuser/Dockerfile]: Thanks for the fix!

**Review by vivian-rook — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**